src/diagonal.works/b6/ingest/yaml_test.go

```diff
@@ -1,20 +1,6 @@
 package ingest
 
-import (
-	"bytes"
-	"fmt"
-	"math"
-	"testing"
-
-	"diagonal.works/b6"
-	"diagonal.works/b6/osm"
-	"diagonal.works/b6/test"
-	"github.com/golang/geo/s1"
-	"github.com/golang/geo/s2"
-	"github.com/google/go-cmp/cmp"
-	"google.golang.org/protobuf/testing/protocmp"
-)
-
+/* // m: fix expression comparing
```

Comment:
I assume you are going to fix this before submission? If not, you can use t.Skip()

Reply:
skip ?? do u even know me ?? no thx, i d rather spend my sunday night looking for the offender, which was, surprise, equal on a pointer, again, fml, but all perfect now